docs/sources/flow/reference/cli/convert.md

```diff
@@ -0,0 +1,32 @@
+---
+title: grafana-agent convert
+weight: 100
+---
+
+# `grafana-agent convert` command
+
+The `grafana-agent convert` command converts a supported configuration file format
+to a Grafana Agent Flow river file.
+
+## Usage
+
+Usage: `grafana-agent convert [FLAG ...] FILE_NAME`
+
+If the `FILE_NAME` argument is not provided or if the `FILE_NAME` argument is
+equal to `-`, `grafana-agent convert` converts the contents of standard input. Otherwise,
+`grafana-agent convert` reads and converts the file from disk specified by the argument.
+
+The `--output` flag can be specified to write the contents of the converted
+file on disk to the specified path. `--output` can only be provided when
+`grafana-agent convert` is not reading from standard input.
+
+The command fails if the file being converted has syntactically incorrect
+configuration or cannot be converted to a Grafana Agent Flow river file.
```

Comment:
nit: Should both uses of file be changed to config to handle stdin/stdout

Reply:
Can you elaborate on this 1 or was it addressed with my last commit? I'm not sure what it is referring to in the context of this paragraph.

Reply:
IE the input can be stdin and output can be stdout and not just files. 

```
The command fails if the config being converted has syntactically incorrect
configuration or cannot be converted to a Grafana Agent Flow river config.
```

Its a super minor opinion and if you think file is clearer keep it.

Reply:
got it now, thanks for explaining. I'm updating throughout for my next pass with extra docs.